pkg/daemon/daemon.go

```diff
@@ -599,7 +610,7 @@ func (dn *Daemon) completeDrain() error {
 		}
 	}
 
-	if dn.openshiftContext.IsOpenshiftCluster() && !dn.openshiftContext.IsHypershift() {
+	if dn.openshiftContext.IsOpenshiftCluster() && !dn.openshiftContext.IsHypershift() && dn.isMcpPaused() {
```

Comment:
i dont think you can reach in openshift a state where you need to complete the drain and mcp is not paused.
because  u will only drain after calling PauseMCP

Reply:
it's just a safe check here :)

because the completeDrain gets called `if dn.isNodeDraining()` that on OCP can be both `Draining` or `MCP_PAUSE` labels so I just wanted to add another safe point

Reply:
do you think its needed ? is there a scenario in mind where this might happen ?